src/backend/distributed/executor/multi_utility.c

```diff
@@ -76,6 +77,7 @@
 #include "utils/palloc.h"
 #include "utils/rel.h"
 #include "utils/relcache.h"
+#include "utils/snapmgr.h"
```

Comment:
I previously had push/pop snapshot calls in here. Removed, so I'll remove this `#include`.